pkg/apis/registry/v1alpha3/types.go

```diff
@@ -66,6 +69,19 @@ type GarbageCollection struct {
 	TTL metav1.Duration `json:"ttl"`
 }
 
+// Proxy contains settings for a proxy used in the registry cache.
+type Proxy struct {
+	// HTTPProxy is used as HTTP_PROXY env in the StatefulSet.
+	// +optional
+	HTTPProxy *string `json:"httpProxy"`
+	// HTTPSProxy is used as HTTPS_PROXY env in the StatefulSet.
+	// +optional
+	HTTPSProxy *string `json:"httpsProxy"`
+	// NoProxy is used as NO_PROXY env in the StatefulSet.
+	// +optional
+	NoProxy *string `json:"noProxy"`
+}
```

Comment:
Missing `,omitempty` for an optional field.

```suggestion
	// HTTPProxy is used as HTTP_PROXY env in the StatefulSet.
	// +optional
	HTTPProxy *string `json:"httpProxy,omitempty"`
	// HTTPSProxy is used as HTTPS_PROXY env in the StatefulSet.
	// +optional
	HTTPSProxy *string `json:"httpsProxy,omitempty"`
	// NoProxy is used as NO_PROXY env in the StatefulSet.
	// +optional
	NoProxy *string `json:"noProxy,omitempty"`
```

Reply:
fixed